simulation/tests/test_costs.py

```diff
@@ -210,10 +217,23 @@ def test_boiler_upgrade_scheme_grant_is_zero_when_grant_cap_exceeded(
         model = model_factory(
             start_datetime=datetime.datetime(2023, 1, 1, 0, 0),
         )
-        model.boiler_upgrade_scheme_cumulative_spend_gbp = 3_000_000
+
+        num_households = random.randint(0, 5)
```

Comment:
Think we need to change this to prevent a division by zero below.
```suggestion
        num_households = random.randint(1, 5)
```